### ISSUE TITLE
Delist portal; Bouncing your confirmation email (500 5.0.0: Line too long) RFC2822

### ISSUE THREAD

**Issue by sietearistas** [MicrosoftDocs/microsoft-365-docs#712]
Hi,
Your Delist Portal is sending a confirmation email that doesn't conform https://tools.ietf.org/html/rfc2822#section-2.1.1
** smtp-in: session [] received invalid input: 500 5.0.0: Line too long ** 
Will appreciate it if you could follow the RFCs and fix this.
Thanks in advance.

**Comment by mypil** [comment]
@sietearistas - May we know which Microsoft document link you used in reference to this issue?
Please share it with us so we can investigate further. Thank you.

**Comment by sietearistas** [comment]
@mypil Thanks your reply
(AS16012609) [DB5EUR01FT053.eop-EUR01.prod.protection.outlook.com]

**Comment by mypil** [comment]
@sietearistas - We need the Microsoft document link so we can investigate. The information you provided appears to be in your outlook. Please understand that this site is a product documentation site and not a support nor a feature request site. As such, the feedback section here is intended for feedback that improves the content in the Microsoft documentation and not intended for support type issues or user feedback on product features.

If you have technical issues, then it would be best for you to contact technical support and open a support ticket to get immediate resolution. Please follow this link to contact support for business products:  https://docs.microsoft.com/office365/admin/contact-support-for-business-products

However, if in case you find that it is a documentation error, feel free to comment back here and we'll be glad to help fix it. Thank you!

**Comment by sietearistas** [comment]
@mypil  I'm afraid I can't provide you with what you call a "microsoft document link" because I'm not an Office365 user.

I'm merely supervising a client's mail server who happens to have difficulties sending emails to some of your Office365 users, and who received a notification informing them that their ip address was *banned*.
In that same notification, my client was instructed to check the following page
https://docs.microsoft.com/en-us/microsoft-365/security/office-365-security/use-the-delist-portal-to-remove-yourself-from-the-office-365-blocked-senders-lis
and use the
https://sender.office.com/
portal to solve this problem.

If this repository doesn't address issues regarding this portal, it might be necessary to change the wording of the notification sent by the mailer-daemon, because the links many people followed to complain about this service all landed here at
https://github.com/MicrosoftDocs/microsoft-365-docs

During the delisting process at https://sender.office.com/ the user is asked to pair their "offending" ip address with their email account (1), and expect a confirmation email.

As I've seen confirmed by other users on this board, this email confirmation fails sometimes (2) to reach the user's inbox, failing to complete the process.
I opened this issue in an effort to help others identify which might be the problem.
Unless you have access to your mail server logs this might go unnoticed, and the incoming **malformed email** will be rejected, usually without any further warning.

I hope this clarifies the issue now.

(1) Ironically, as pointed out by @bcebulla https://github.com/MicrosoftDocs/microsoft-365-docs/issues/592#issuecomment-550073822
it seems that the Microsoft Delisting Service doesn't make any kind of MX-record checking about the alleged banned ip address, and the mail address provided.
Long story short, for the time being, you can use any mail provider to delist yourself.
(2) It does work using more "relaxed" non-RFC-strict mail servers.

**Comment by mypil** [comment]
@sietearistas - Thank you for taking the time to share your assessment and for providing more details about this issue.

I will get this issue over to the Microsoft 365 writing team for investigation.


I made a note to request the team to update this when the work is complete.

**Comment by mypil** [comment]
@sietearistas - We have already engaged with the author and the product engineering team to look into the Delist portal issue. I will close this issue now but please refer to issue #592 on updates on the status. 

Thank you for your contribution to make the docs better! We truly appreciate it.
